examples/src/examples/graphics/model-outline.tsx

```diff
@@ -23,6 +23,10 @@ class ModelOutlineExample {
 
             app.scene.ambientLight = new pc.Color(0.2, 0.2, 0.2);
 
+            // adjust default clustered lighting parameters to handle many lights
+            const lighting = app.scene.lighting;
+            lighting.shadowAtlasResolution = 4096;
```

Comment:
all devices go to 4096?

Reply:
yep. internally we max it to max as well.

Reply:
should we add this setting to the editor scene settings?

Reply:
yep, all clustered lighting settings will be exposed.